src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/AdditionConverter.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class AdditionConverter : IMultiValueConverter
```

Comment:
Can you add Summary explaining the purpose of this converter and what are values[0].. values[2]?